models/pancreas_ct_dints_segmentation/docs/README.md

```diff
@@ -4,9 +4,9 @@ A neural architecture search algorithm for volumetric (3D) segmentation of the p
 ![image](https://developer.download.nvidia.com/assets/Clara/Images/clara_pt_net_arch_search_segmentation_workflow_4-1.png)
 
 ## Data
-The training dataset is the Panceas Task from the Medical Segmentation Decathalon. Users can find more details on the datasets at http://medicaldecathlon.com/.
+The training dataset is the Pancreas Task from the Medical Segmentation Decathalon. Users can find more details on the datasets at http://medicaldecathlon.com/.
 
-- Target: Liver and tumour
+- Target: Pancreas and pancreatic tumor
```

Comment:
Hi @dongyang0122 , could you please help to double confirm the changes of this bundle? Thanks in advance!

Reply:
The adjustments made to the README are accurate. Thanks!